### PR TITLE
Refatora pipeline do GTFS

### PR DESCRIPTION
**Pull request by akaBotelho**

### CODE REVIEW

File: pipelines/tasks.py [hunk 1]
```diff
@@ -214,3 +216,11 @@ def run_subflow(
 
     if flag_failed_runs:
         raise FailedSubFlow(failed_message)
+
+
+@task(trigger=all_finished)
+def check_fail(results):
```
Comment (pixuimpou) [inline pipelines/tasks.py:222]: Adicionar docsting

File: queries/profiles.yml [hunk 2]
```diff
@@ -78,4 +78,4 @@ queries:
             spark.executor.instances: "2"
             spark.driver.memory: 4g
             spark.driver.memoryOverhead: 1g
-  target: prod
\ No newline at end of file
+  target: hmg
```
Comment (pixuimpou) [inline queries/profiles.yml:81]: mudar pra prod

File: pipelines/migration/tasks.py [hunk 3]
```diff
@@ -1588,6 +1585,142 @@ def transform_raw_to_nested_structure(
...
+            # filepath = save_treated_local_func(data=data, error=error, filepath=filepath)
+
+        except Exception:  # pylint: disable=W0703
+            error = traceback.format_exc()
+            log(f"[CATCHED] Task failed with error: \n{error}", level="error")
+
+    return error, filepath
+
+
+# @task(nout=2)
```
Comment (pixuimpou) [inline pipelines/migration/tasks.py:1679]: remover comentarios

File: pipelines/migration/br_rj_riodejaneiro_gtfs/flows.py [hunk 4]
```diff
@@ -239,10 +300,10 @@
 gtfs_captura_nova.run_config = KubernetesRun(
     image=constants.DOCKER_IMAGE.value,
     labels=[constants.RJ_SMTR_AGENT_LABEL.value],
-    cpu_limit="1000m",
-    memory_limit="4600Mi",
-    cpu_request="500m",
-    memory_request="1000Mi",
+    # cpu_limit="1000m",
```
Comment (pixuimpou) [inline pipelines/migration/br_rj_riodejaneiro_gtfs/flows.py:303]: remover comentarios

**Review by vtr363 — DISMISSED** [review]

**Pull request merged**